module_utils/network_lsr/argument_validator.py

```diff
@@ -1252,6 +1252,11 @@ def __init__(self):
                 ArgValidatorDeprecated("master", deprecated_by="controller"),
                 ArgValidatorStr("interface_name", allow_empty=True),
                 ArgValidatorMac("mac"),
+                ArgValidatorList(
+                    "interface_path",
+                    nested=ArgValidatorStr("interface_path[?]"),
+                    default_value=list,
```

Comment:
why is the default value an empty list? Just have it `None`. There is a difference between having a parameter unset and set to an empty list.

Well, maybe you'd like to treat them the same or reject an empty list as invalid, but as far as the normalization/validation of `ArgValidator` is concerned, it should not loose this information.

Reply:
I set the default value for `interface_path` to be `None`